### DISCUSSION TITLE
Coding style guideline is added in July 2024

### DISCUSSION THREAD

**Discussion by yungyuc** [solvcon/modmesh#395] [General]
We had some good discussions in the PR https://github.com/solvcon/modmesh/pull/394#discussion_r1689474905. I created this conversation for further discussions on the coding style in modmesh.

**Comment by ThreeMonth03** [comment]
I'm late for reviewing PR https://github.com/solvcon/modmesh/pull/394#discussion_r1689474905, and I have a quick question. Should we highlight the [code block of copyright notice](https://github.com/solvcon/modmesh/blob/master/STYLE.rst?plain=1#L451) according to the programming language?

**Reply by yungyuc** [reply]
Good catch.  While it's not critical, I think we can.  Feel free to file a PR for it.

**Reply by ThreeMonth03** [reply]
PR #409 add the syntax highlighting for all code blocks.
